gnovm/tests/files/map0.gno

```diff
@@ -8,8 +8,5 @@ func main() {
 	}
 }
 
-// Error:
-// duplicate key "hello" in map literal
-
 // TypeCheckError:
 // main/map0.gno:7:3: duplicate key "hello" in map literal; main/map0.gno:4:2: declared and not used: m
```

Comment:
this should fail as it were:
https://go.dev/play/p/WynhspMPEKf

Reply:
this should fail in preprocess/typecheck not on runtime. So if it fails in typecheck, I think we said we don't need to check the error on the preprocess part.

Reply:
can you try // Output: and print the value?

Reply:
>this should fail in preprocess/typecheck like on golang not on runtime(here we have same error). So if it fails in typecheck, I think we said we don't need to check the error on the preprocess part.

yeah but i think Gno should also behave the same way